### ISSUE TITLE
bootstrap?

### ISSUE THREAD

**Issue by zztop101** [FountainJS/generator-fountain-webapp#43]
Sorry, maybe due to my lack of understanding of webpack - but how do I include bootstrap CSS/SASS into my project, and ng2-bootstrap as well ?

**Comment by zckrs** [comment]
Hi,

Not directly related to our repositories but you should ng2-bootstrap with npm `npm install ng2-bootstrap --save`

And use import in your class.

IMO you should find more information to [README of ng2-bootstrap](https://github.com/valor-software/ng2-bootstrap)

**Comment by zztop101** [comment]
Hi, the problem I found was ng2-bootstrap suggests using the CDN version of bootstrap for the css, but I wanted a local install of the css. I went down the path of using Sass and importing from bootstrap-sass which lead to issues with the glyphicons and I had to do multiple things to get it working. Later I found I could just install bootstrap and from the index.ts I could import 'bootstrap/dist/css/bootstrap.css';
The point I want to make is it took a couple of hours to get bootstrap and ng2-bootstrap working in my project and I am sure other people will have the same issue. gulp-angular generator set up bootstrap and angularUI for me.
